### DISCUSSION TITLE
403 Forbidden on `GET /sellers/v1/account`

### DISCUSSION THREAD

**Discussion by AJ-AUS** [amzn/selling-partner-api-models#4342] [Q&A]
#### Issue
Calling `GET /sellers/v1/account` always returns a 403 Forbidden: "Access to the requested resource is denied."

#### What Works
- `GET /sellers/v1/marketplaceParticipations` works fine with the same setup.

#### What Doesn’t Work
- `GET /sellers/v1/account` fails in all regions and environments (Sandbox, Production).

#### Note from Docs
- Docs mention `GET /sellers/v1/account` is **only available in Sandbox** and **only in the EU marketplace**. Could this be the issue?

#### Additional Information
- I can confirm we have access to **Finance and Accounting role** in our developer profile.

Has anyone gotten this to work?

**Comment by hakanensari** [comment]
I also ran into this a while ago, and the developer support team confirmed the limited scope:

> The 'getAccounts' operation is only available for Sandbox endpoints and is limited to the EU marketplace. Therefore, it is expected that you will encounter an Error 403 when attempting to use this operation in a production environment.

**Reply by AJ-AUS** [reply]
Thanks @hakanensari  

Unfortunately, I am getting 403 in Sandbox as well.

**Reply by mafge** [reply]
Hi @AJ-AUS,
unfortunately the sandbox call to get account information is broken at the moment. The team is aware of it and will fix it as soon as possible.
Best,
Marc
Selling Partner Developer Services
